### PR TITLE
chapter two unedited

### PR DESCRIPTION
**Pull request by nikymorg**
I didn't read the synopsis and wouldn't let @realAndrewCohn read his chapter to me first, but I remain confident that this is in keeping with those both with respect to plot and overall tone.

### CODE REVIEW

File: chapter-2.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Cold and naked was not what Dicky liked to be. But cold and naked was what Dicky was, and so he resigned himself to it. The rough and ridged sheet metal of the boxcar floor created and interesting and not unpleasant sensation when Dicky scooted across it to open the boxcar door. He looked forward to spending long hours searching the web for tetanus symptoms and applying them to himself later. He would hopefully be clothed by then. And maybe warm too, in a cozy internet cafe with a steel-haired waitress named Loretta who took neither flack nor guff. But in order to get to then, he had to leave the now. And leaving the now meant jumping out of a moving train. Oh well, he thought. Here goes Dicky!
```
Comment (realAndrewCohn) [inline chapter-2.md]: `The rough and ridged sheet metal of the boxcar floor created and interesting and not unpleasant sensation when Dicky scooted across it to open the boxcar door`

`created and interesting`

`and interesting`

`and`

**Comment by nikymorg** [comment]
@RachelSa pls advise how I can add reviewers to my pr. Advise pls.

**Comment by nikymorg** [comment]
Worried this chapter may draw too heavily on the children's book series 'The Boxcar Kids' not that I remember any of it but there _was_ a boxcar. But I suppose if you have read one romantic, murderous, dystopian YA novel you have read them all.

**Comment by realAndrewCohn** [comment]
@nikymorg i disagree. I think the cold and nakedness really add a layer of complexity to this already deeply layered novel. 5 stars

**Comment by RachelSa** [comment]
wowowowo. DC crew is #talent. @nikymorg the fact that you did not read the prompt shows your strong, strong artistic vision.

**Pull request merged**